homework_11/def_temperature.py

```diff
@@ -0,0 +1,27 @@
+"""
+Задача: Реалізуйте функцію, яка конвертує температуру з градусів Цельсія в градуси Фаренгейта і навпаки.
+Функція повинна приймати два аргументи: значення температури і одиницю виміру ('C' або 'F') і повертати перетворене значення.
+
+Фарингейт = (градус Цельсий * 1.8) + 32
+Цельсий = 0.5556 * (градус Фарингейт - 32)
+"""
+FAHRENHEIT_COEFFICIENT = 1.8
+CELSIUS_COEFFICIENT = 0.5556
+
+
+def convert_temperature(*, temp_value: int, unit: str) -> str:
+    # From Celsius to Fahrenheit
+    if unit.upper() == 'C':
+        result_fahrenheit = (temp_value * FAHRENHEIT_COEFFICIENT) + 32
+        return f'{temp_value} celsius is equal {result_fahrenheit} fahrenheit'
+    # From Fahrenheit to Celsius
+    elif unit.upper() == 'F':
+        result_celsius = CELSIUS_COEFFICIENT * (temp_value - 32)
+        return f'{temp_value} fahrenheit is equal {result_celsius} celsius'
+    else:
+        raise ValueError(f'\'{unit}\' is not a valid unit. You should choose one of these \'C\' or \'F\'')
+
```

Comment:
One more variant
![image](https://github.com/SerhiiKotsar/hillel_python_basic/assets/51053483/29959cd4-3df6-4bcc-ae6a-f942d25e1d66)

Reply:
One more
![image](https://github.com/SerhiiKotsar/hillel_python_basic/assets/51053483/0b4e5876-9eeb-4364-bca0-ca4e6c02a944)

Reply:
thank you